packages/client/src/v2-events/hooks/useUserDetails.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+
+import { useSelector } from 'react-redux'
+import { getLocations } from '@client/offline/selectors'
+// eslint-disable-next-line no-restricted-imports
+import { getUserDetails } from '@client/profile/profileSelectors'
+
+export function useUserDetails() {
```

Comment:
```suggestion
export function useUserAddress() {
```
seems more appropriate no?

Reply:
Yes!